src/main/java/de/utkast/sputnik/SputnikTest.java

```diff
@@ -0,0 +1,19 @@
+package de.utkast.sputnik;
+
+public class SputnikTest {
+
+	// findbugs warning
```

Comment:
\[Checkstyle\] ERROR: File contains tab characters \(this is the first instance\)\.